packages/mobile/src/components/details-tile/DetailsTileNoAccess.tsx

```diff
@@ -1,24 +1,109 @@
+  Chain,
+  useSpecialAccessEntity,
+  premiumContentSelectors
+} from '@audius/common'
+import { View, Text, Image } from 'react-native'
 import { useSelector } from 'react-redux'
 
+import IconLock from 'app/assets/images/iconLock.svg'
+import { useLink } from 'app/components/core'
+import UserBadges from 'app/components/user-badges'
+import { flexRowCentered, makeStyles } from 'app/styles'
+import { useColor } from 'app/utils/theme'
+
 const { getPremiumTrackStatusMap } = premiumContentSelectors
 
-const DetailsTileLockedSection = ({
-  premiumConditions
-}: {
-  premiumConditions: PremiumConditions
-}) => {
-  return <View></View>
+const messages = {
+  unlocking: 'UNLOCKING',
+  howToUnlock: 'HOW TO UNLOCK',
+  lockedCollectibleGated:
+    'To unlock this track, you must link a wallet containing a collectible from:',
+  unlockingCollectibleGatedPrefix: 'A Collectible from ',
+  unlockingCollectibleGatedSuffix: ' was found in a linked wallet.',
+  lockedFollowGatedPrefix: 'Follow ',
+  unlockingFollowGatedPrefix: 'Thank you for following ',
+  unlockingFollowGatedSuffix: ' !',
+  lockedTipGatedPrefix: 'Send ',
+  lockedTipGatedSuffix: ' a tip.',
+  unlockingTipGatedPrefix: 'Thank you for supporting ',
+  unlockingTipGatedSuffix: ' by sending them a tip!'
 }
 
-const DetailsTileUnlockingSection = ({
-  premiumConditions
-}: {
-  premiumConditions: PremiumConditions
-}) => {
-  return <View></View>
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  root: {
+    marginBottom: spacing(4),
+    padding: spacing(4),
+    backgroundColor: palette.neutralLight10,
+    borderWidth: 1,
+    borderColor: palette.neutralLight7,
+    borderRadius: spacing(2)
+  },
+  titleContainer: {
+    ...flexRowCentered(),
+    marginBottom: spacing(2)
+  },
+  title: {
+    marginLeft: spacing(2),
+    fontFamily: typography.fontByWeight.heavy,
+    fontSize: typography.fontSize.medium,
+    color: palette.neutral
+  },
+  descriptionContainer: {
+    ...flexRowCentered(),
+    flexWrap: 'wrap'
+  },
+  description: {
+    flexShrink: 0,
+    fontFamily: typography.fontByWeight.demiBold,
+    fontSize: typography.fontSize.medium,
+    color: palette.neutral,
+    lineHeight: spacing(6)
+  },
+  collectionName: {
+    color: palette.secondary
+  },
+  collectionContainer: {
+    ...flexRowCentered(),
+    marginTop: spacing(2)
+  },
+  collectionImage: {
+    marginRight: spacing(2),
+    borderWidth: 1,
+    borderColor: palette.neutralLight7,
+    borderRadius: spacing(1),
+    width: spacing(8),
+    height: spacing(8)
+  }
+}))
+
+type NoAccessProps = {
+  renderDescription: () => ReactNode
+  isUnlocking: boolean
+}
+
+const DetailsTileNoAccessSection = ({
+  renderDescription,
+  isUnlocking
+}: NoAccessProps) => {
+  const styles = useStyles()
+  const neutral = useColor('neutral')
+
+  return (
+    <View style={styles.root}>
+      <View style={styles.titleContainer}>
+        <IconLock fill={neutral} width={16} height={16} />
```

Comment:
Just double-checking that design wants this to always be `neutral` color regardless of theme?